service/cloudconfig/v_0_1_0_master_template.go

```diff
@@ -137,6 +140,12 @@ func (e *v_0_1_0MasterExtension) Files() ([]k8scloudconfig.FileAsset, error) {
 			Owner:        FileOwner,
 			Permissions:  FilePermission,
 		},
+		{
+			AssetContent: e.keys.APIServerEncryptionKey,
+			Path:         "/etc/kubernetes/encryption/k8s-encryption-config.yaml.enc",
```

Comment:
as far as i understand this file contains NOT encrypted base64 encoded key in yaml. So no need for `.enc` extension.

Reply:
Good catch. Copy/Paste error